test/run.bats

```diff
@@ -358,3 +358,17 @@ EOF
     done
 }
 
+@test 'resolves relative path to image' {
+   DIRNAME=`dirname $CHTEST_IMG`
+   BASEDIR=`basename $CHTEST_IMG`
+   (
+      cd $DIRNAME
+      ch-run $BASEDIR -- true
+   )
+}
+
+@test 'resolves symbolic link to image' {
+   ln -s $CHTEST_IMG $IMGDIR/symlink-test
```

Comment:
Here, use `$BATS_TMPDIR` instead of `$IMGDIR` because other parts of the test suite can get cranky if unknown things are found in the latter (e.g., try `make test-quick` twice in a row). Then you also don't need to remove it later.